src/features/user/CompleteSignup/index.tsx

```diff
@@ -3,20 +3,19 @@ import { useRouter } from 'next/router';
 import styles from './CompleteSignup.module.scss';
 import MaterialTextField from '../../common/InputTypes/MaterialTextField';
 import ToggleSwitch from '../../common/InputTypes/ToggleSwitch';
-import Snackbar from '@material-ui/core/Snackbar';
-import MuiAlert from '@material-ui/lab/Alert';
+import { Snackbar,Alert as MuiAlert, MenuItem } from '@mui/material';
```

Comment:
Snackbar - couldn't see the snackbar. When does it appear?
MenuItem - slight font size/font style differences compared to develop brancg

Reply:
@Shreyaschorge - when does the snackbar appear? Font related issues are resolved.

Reply:
> Snackbar - couldn't see the snackbar. When does it appear? 

@Shreyaschorge Snackbar isn't seen.

> MenuItem - slight font size/font style differences compared to develop brancg
Resolved in #1467